unittests/FEXLinuxTests/tests/signal/sigtest_defer.cpp

```diff
@@ -0,0 +1,58 @@
+// A test where a signal is masked, a value is set that the signal handler will overwrite, and then the signal is unmasked to allow it to
+// fire. FEX-Emu had a bug where it wasn't properly deferring signals from sigprocmask if one of the signals was the few that FEX must never
+// block. In older glibc versions (glibc-2.26), the `raise` implementation would block signals, tgkill, and then unblocked signals,
```

Comment:
This sentence is a bit awkward, not sure I understand it correctly but maybe this is better?

```suggestion
// fire. FEX-Emu had a bug where it wasn't properly deferring signals from sigprocmask if one of the signals was masked by the guest application. 
// In older glibc versions (glibc-2.26), the `raise` implementation would block signals, tgkill, and then unblock signals,
```

Reply:
Switched over to the the comment